src/ui/components/Comments/TranscriptComments/types.ts

```diff
@@ -1,7 +1,15 @@
+import React from "react";
 import { Comment, Reply } from "ui/state/comments";
 
 export type CommentData =
   | { type: "comment"; comment: Comment }
   | { type: "new_comment"; comment: Comment }
   | { type: "reply"; comment: Reply }
   | { type: "new_reply"; comment: Reply };
+
+export const FocusContext = React.createContext({
+  autofocus: false,
+  isFocused: false,
+  blur: () => {},
+  close: () => {},
+});
```

Comment:
I don't think the `FocusContext` belongs in a file named `types`. Make it it's own file?

Reply:
Yeah, I can do that - I just saw that it was a non-component file that wasn't part of any circles yet and figured it was a decent place to move the context declaration.